src/Blocks/components/utils/index.js

```diff
@@ -332,14 +333,25 @@ export const getSettingsJsonOptions = (options, useEmpty = false) => {
  *
  * @returns Component
  */
-export const MissingName = ({ value, asPlaceholder, className = '' }) => {
+export const MissingName = ({
+	value,
+	asPlaceholder,
+	className = '',
+	isOptional = false,
+}) => {
 	if (value || asPlaceholder) {
 		return null;
 	}
 
+	const style = classnames([
+		'es-position-absolute es-right-0 es-top-0 es-nested-color-pure-white es-nested-w-5 es-nested-h-5 es-w-8 es-h-8 es-rounded-full es-has-enhanced-contrast-icon es-display-flex es-items-center es-content-center',
+		!isOptional ? 'es-bg-red-500' : 'es-bg-yellow-500',
+		selector(className, className),
+	]);
```

Comment:
```suggestion
	const style = classnames(
		'es-position-absolute es-right-0 es-top-0 es-nested-color-pure-white es-nested-w-5 es-nested-h-5 es-w-8 es-h-8 es-rounded-full es-has-enhanced-contrast-icon es-display-flex es-items-center es-content-center',
		isOptional ? 'es-bg-yellow-500' : 'es-bg-red-500',
		className,
	);
```
The function handles most of the stuff for ya 😉